extensions/wikia/PowerUser/PowerUser.setup.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Manages data on wikia's PowerUsers
+ *
+ * @package Wikia\extensions\PowerUser
+ * @version 1.0
+ * @author Adam Karminski <adamk@wikia-inc.com>
+ * @see https://github.com/Wikia/app/tree/dev/extensions/wikia/PowerUser/
+ */
+
+/**
+ * @global Array $wgExtensionCredits The list of extension credits.
+ * @see http://www.mediawiki.org/wiki/Manual:$wgExtensionCredits
+ */
+$wgExtensionCredits['other'][] = array(
```

Comment:
We've standardised on the [short array syntax](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax).